Gemfile

```
@@ -20,32 +20,30 @@ gem 'jbuilder', '~> 2.7.0'
 gem 'nokogiri', '>= 1.8.2'
 gem 'puma', '~> 5.5.1'
 gem 'rack', '~> 2.2.3'
-gem 'rails', '~> 5.2'
+gem 'rails'
 gem 'rake'
-# gem 'sprockets', '~> 3.7.0', '>= 3.7.2'
 
 gem 'pg'
-gem 'sequel'
+gem 'sequel', '~> 5.51.0'
```

Is it really necessary? If so let's stick to major:
```suggestion
gem 'sequel', '~> 5.51'
```